util/queue.py

```diff
@@ -66,7 +66,7 @@ class Queue(multiprocessing.queues.Queue):
     """
 
     def __init__(self, *args, **kwargs):
-        super(Queue, self).__init__(*args, **kwargs)
+        super(Queue, self).__init__(*args, **kwargs, ctx=multiprocessing.get_context())
```

Comment:
This method is new in Python 3.4. You should check the Python version and add it to the list of parameters only if Python >= 3.4. For example:

```
def __init__(self, *args, **kwargs):
    if  sys.version_info >= (3, 4) and 'ctx' not in kwargs:
        kwargs['ctx'] = multiprocessing.get_context()
    super(Queue, self).__init__(*args, **kwargs)
```